src/isomorphic/classic/class/ReactClass.js

```diff
@@ -9,6 +9,8 @@
  * @providesModule ReactClass
  */
 
+/* global hasOwnProperty:true */
```

Comment:
Such a comment was in the files where hasOwnProperty was already used before my fixes. You can delete this comment.